Uchu.World/Objects/Components/ReplicaComponents/MovingPlatformComponent.cs

```diff
@@ -34,7 +34,18 @@ public class MovingPlatformComponent : ReplicaComponent
 
         public uint NextWaypointIndex { get; set; }
 
-        public float IdleTimeElapsed { get; set; }
+        /// <summary>
+        ///     Time spent idle
+        /// </summary>
+        public float IdleTimeElapsed =>
+            State == PlatformState.Idle
+                ? (float) ((DateTime.UtcNow - new DateTime(1970, 1, 1, 0, 0, 0)).TotalSeconds - _wayPointStartTime)
```

Comment:
Could you elaborate a bit on why you're subtracting UtcNow with a date in the past instead of just subtracting two dates? Calling DateTime.UtcNow multiple times should provide a new accurate date each time that doesn't update anymore afterwards.

Reply:
I had it in my mind of using the normal Unix timestamp for comparisons and didn't think that UtcNow without that step is fine in this context since the start point is arbitrary.

Reply:
Can't you just use DateTime.UtcNow twice, then subtract them, and then do total seconds?